src/main/java/frc/team2412/robot/Hardware.java

```diff
@@ -27,6 +31,24 @@ public class Hardware {
 	public static final Rotation2d DRIVEBASE_BACK_RIGHT_ENCODER_OFFSET =
 			Rotation2d.fromDegrees(311.396);
 	public static final int GYRO_PORT = 13;
+
+	// Cameras
+	public static final String PHOTON_CAM = "OV5647";
+	// Rough measurements, origin is center of robot, +X is forward, +Y is left, +Z is up
+	public static final Transform3d ROBOT_TO_CAM =
+			new Transform3d(
+					new Translation3d(
+							// 7 inches from back of robot, back is -half of length (30 in.)
+							Units.inchesToMeters(-30.0 / 2 + 7),
+							// 7 inches from left, left is +half of width (26 in.)
+							Units.inchesToMeters(26.0 / 2 - 7),
+							// 30 inches above the ground
+							Units.inchesToMeters(30)),
+					// Camera has a slight yaw, around -0.25 radians following right-hand rule (thumb points
+					// to +Z/up, fingers curl in positive rotation)
+					new Rotation3d(0, 0, -0.25));
```

Comment:
It's be more human-understandable to use a degrees constant & convert to radians for code

Reply:
fixed